src/DataStructures/VectorImpl.hpp

```diff
@@ -591,3 +591,52 @@ std::ostream& operator<<(std::ostream& os,
                    });                                                  \
     return ret;                                                         \
   }
+
+/*!
+ * \brief Helper struct to determine the base storage type of a vector or
+ * container
+ *
+ * \details in member type `type` is stored the base stored type of an object
```

Comment:
I find this sentence confusing as I don't understand `is stored` and the word `type` is used too often